src/templates/ja/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+        </p>
+
+        <div class="code-block floating-card">
+          <pre><code><span class="code-comment">#standardSQL
+# 01_01b: Distribution of JS bytes by client</span>
+<span class="code-keyword">SELECT</span>
+  percentile,
+  _TABLE_SUFFIX <span class="code-keyword">AS</span> client,
+  <span class="code-function">APPROX_QUANTILES</span>(<span class="code-function">ROUND</span>(bytesJs <span class="code-keyword">/</span> <span class="code-number">1024</span>, <span class="code-number">2</span>), <span class="code-number">1000</span>)[<span class="code-function">OFFSET</span>(percentile <span class="code-keyword">*</span> <span class="code-number">10</span>)] <span class="code-keyword">AS</span> js_kbytes
+<span class="code-keyword">FROM</span>
+  <span class="code-string">`httparchive.summary_pages.2019_07_01_*`</span>,
+  <span class="code-function">UNNEST</span>([<span class="code-number">10</span>, <span class="code-number">25</span>, <span class="code-number">50</span>, <span class="code-number">75</span>, <span class="code-number">90</span>]) <span class="code-keyword">AS</span> percentile
+<span class="code-keyword">GROUP BY</span>
+  percentile,
+  client
+<span class="code-keyword">ORDER BY</span>
+  percentile,
+  client</code></pre>
+        </div>
+
+        <p>
+          各指標の結果は、<a href="https://docs.google.com/spreadsheets/d/1kBTglETN_V9UjKqK_EFmFjRexJnQOmLLr-I2Tkotvic/edit?usp=sharing">JavaScriptの結果</a>のように、章ごとのスプレッドシートで公開されています。
+        </p>
+    </section>
+
+    <section>
+        <h3 id="websites"><a href="#websites" class="anchor-link">ウェブサイト</a></h3>
+
+        <p>
+          データセットには7,546,709件のウェブサイトが含まれている。そのうち、6,347,919件がモバイルサイト、5,593,642件がデスクトップサイトである。ほとんどのウェブサイトがモバイルとデスクトップの両方のサブセットに含まれている。
+        </p>
+
+        <p>
+          HTTP Archiveは、<a href="#chrome-ux-report">Chrome UXレポート</a> からウェブサイトのURLを取得しています。Chrome UXレポートはGoogleが公開しているデータセットで、Chromeユーザーが積極的に訪問している数百万のウェブサイトのユーザー体験を集計しています。これにより、実際のウェブ利用状況を反映した最新のウェブサイトのリストが得られます。Chrome UXレポートデータセットにはフォーム ファクター ディメンションが含まれており、デスクトップ ユーザーやモバイル ユーザーがアクセスしたすべてのWebサイトを取得するために使用します。
+        </p>
+
+        <p>
+          Web Almanacが使用した2020年8月のHTTP Archiveのクロールでは、Webサイトのリストに、最新のChrome UXレポートリリースである2020年6月（202006）を使用しました。このデータセットは2020年7月14日にリリースされたもので、6月中にChromeユーザーが訪問したウェブサイトをキャプチャしています。
+        </p>
+
+        <!-- To be translated <p>
+          There was was a 20-30% growth on websites analysed compared to the <a href="../2019/methodology#websites">number of sites analysed for the 2019 Web Almanac</a>. This increase has been analysed by <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}">Paul Calvano</a> in his <a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/">Growth of the Web in 2020</a> post.
+        </p> -->
```

Comment:
Can a native speaker translate this whole section and remove the `<!--` and `-->` marks?

Reply:
Yes, I think can.

Reply:
```suggestion
<p>2019年のWeb Almanac</a>のために分析した<a href="../2019/methodology#websites">サイト数と比較して、分析したウェブサイトで20～30%の増加がありました。この増加は、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}">Paul Calvano</a>が<a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/">Growth of the Web in 2020</a>の投稿で分析したものです。</p>
```